src/coreclr/src/pal/src/libunwind/include/libunwind-aarch64.h

```diff
@@ -184,7 +184,7 @@ struct unw_sigcontext
 	uint64_t sp;
 	uint64_t pc;
 	uint64_t pstate;
-	uint8_t __reserved[(34 * 8)] __attribute__((__aligned__(16)));
+	uint8_t __reserved[(66 * 8)] __attribute__((__aligned__(16)));
```

Comment:
Where was this causing a problem? The comment above says this field was intentionally trimmed down.

Reply:
I pushed the patch upstream.  It merged today as libunwind/libunwind#179
It was trimmed down from 4096.
It needed to be 66 to match unw_fpsimd_t below.